packages/babel-parser/test/fixtures/flow/optional-type/5/options.json

```diff
@@ -1,3 +1,3 @@
 {
-  "throws": "Unexpected token, expected \",\" (1:65)"
+  "throws": "Unexpected token (1:65)"
```

Comment:
The input code is `a ? b : c ? d : e ? f : g ? h : i ? j : k ? l : m ? n : (o ? p : =);`. The old error message was wrong, since there is no way we could expect `,` instead of `=`.